pkg/controller/network/ippool/ippool_controller.go

```diff
@@ -61,38 +65,39 @@ type IPPoolController struct {
 	ippoolSynced   cache.InformerSynced
 	ippoolQueue    workqueue.RateLimitingInterface
 
+	wkInformer tenantv1alpha1informers.WorkspaceInformer
```

Comment:
normally, we use `ws` as an abbreviation for workspace

Reply:
done